pipeline/terraform/main.tf

```diff
@@ -15,6 +15,14 @@ locals {
       scale_up_id_minter_db    = false
       scale_up_matcher_db      = false
     }
+
+    "2023-03-16" = {
```

Comment:
This pipeline has not yet been created.  Adding this stops the automatic deployment putting this code onto the current live pipeline, where it won't work.